test/end-to-end/config.json

```diff
@@ -36,9 +36,10 @@
     },
     "__comment3__": "// Nightmare.js timeout settings",
     "nightmareOptions": {
-      "waitTimeout": 5000,
-      "gotoTimeout": 1000,
-      "loadTimeout": 1000,
-      "executionTimeout": 1000
+      "show": "true",
+      "waitTimeout": 90000,
+      "gotoTimeout": 30000,
+      "loadTimeout": 60000,
+      "executionTimeout": 60000
```

Comment:
OOI, is it only slow the first time when it reads the yum db, or always? In the former case, a better workaround might be to prime the cache before starting the test, with e. g. a curl query to /projects/list ?

Reply:
I'm not sure I understand the question here but the code is not entirely correct either. 

These are the timeouts which worked for me before the 100% CPU load issue was fixed on the backend. I tried playing around with lower values today but couldn't find such that will consistently pass on my local system. 

These timeouts don't have anything to do with testing on lorax-composer though.